app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -232,6 +267,19 @@ void handleMotionEvent(final int aHandle, final int aDevice, final boolean aPres
             public void run() {
                 Widget widget = mWidgets.get(aHandle);
                 MotionEventGenerator.dispatch(widget, aDevice, aPressed, aX, aY);
+
+                // Fixme: Remove this once the new Keyboard delegate lands in GeckoView
```

Comment:
Why did you add this? rebase problem?

Reply:
Yes, I think it was a rebase issue. Fixed.